website/content/docs/configuration/events/common.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: Controller - Events - File Sink - Configuration
+description: |-
+  The common sink parameters for all sink types.
+---
+
+# Common Sink Parameters
+
+These parameters are shared across all sink types:
+
+- `name` - Specify a name for the sink.
+
+- `description` - Specify a description for the sink.
+
+- `event_types` - Specifies a list of event types that will be sent to the sink.
+  Can be `*`, `error`, `system`, `observation` or `audit` 
+
+- `event_source_url` - Specifies an optional event source URL for the sink.  If
+  not specified a default source will be composed of the
+  https://hashicorp.com/boundary.io/ServerName/Path/FileName.
+
+- `allow_filters` - Specifies a set predicates for including an event in the
```

Comment:
There's a whole set of filter docs (https://www.boundaryproject.io/docs/concepts/filtering) and it would probably be good to add a page for event filtering with examples and then link there from here.